### PR TITLE
Stacked area chart

### PR DESCRIPTION
**Pull request by Egor-Watson**

### CODE REVIEW

File: dashframework-main/app.py [hunk 1]
```diff
@@ -50,6 +51,12 @@
 
 simple_barchart = Barchart('accident_year', 'number_of_casualties', grouped)
 
+# Make stacked area chart
+df_merged_area = df_date.join(df_conditions, lsuffix='_date', rsuffix='_conditions')
+grouped_area = df_merged_area.groupby('accident_year').size()
+grouped_area = grouped_area.reset_index(name='count')
+
+stacked_area_chart = StackedAreaChart('accident_year', 'count', 'weather_conditions', None, grouped_area)
```
Comment (Egor-Watson) [inline dashframework-main/app.py:59]: 'weather_conditions' doesn't exist in 'grouped_data'

File: dashframework-main/app.py [hunk 2]
```diff
@@ -126,9 +133,19 @@ def global_filter(year_range, time_range, vehicle_no, start_date, end_date):
     Output(simple_barchart.html_id, 'figure'),
     Input('year-filter-global', 'value') # dummy input, else callback doesn't work
 )
+
+@app.callback(
+    Output('loading-output-2', 'children'), # Loading indicator
```
Comment (Egor-Watson) [inline dashframework-main/app.py:138]: 'loading-out-2' also doesn't exist. Its fine to leave it out for now.

File: dashframework-main/app.py [hunk 3]
```diff
@@ -126,9 +133,19 @@ def global_filter(year_range, time_range, vehicle_no, start_date, end_date):
...
+    Output(stacked_area_chart.html_id, 'figure'),
+    Input('year-filter-global', 'value')
+)
+
 def update_simple_barchart(value):
     return 'Barchart loaded', simple_barchart.update()
 
+def update_stacked_area_chart(value):
```
Comment (Egor-Watson) [inline dashframework-main/app.py:146]: This function needs to be moved just under its callback. So to line 142.

File: dashframework-main/app.py (hunk 1 above)
Comment (dharmsen) [reply dashframework-main/app.py:59]: even when I join with df_condtions? I don't see why this would lead to problems

**Pull request merged**